src/components/auth-form/auth-form-button/auth-form-button.styles.js

```diff
@@ -3,12 +3,12 @@ import { makeStyles } from '@material-ui/core/styles';
 export const useStyles = makeStyles((theme) => ({
   authBtn: {
     fontFamily: theme.typography.fontFamily,
-    background: theme.palette.textColor,
+    backgroundColor: theme.palette.textColor,
     color: theme.palette.backgroundColor,
     fontSize: theme.typography.button.fontSize,
     fontWeight: theme.typography.h2.fontWeight,
     lineHeight: theme.typography.h2.lineHeight,
-    height: '50px',
+    minHeight: '42px',
```

Comment:
Button height should be controlled by paddings and not height property

Reply:
fixed